packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -125,6 +125,58 @@ export interface CloudPulseMetricsList {
   values: [number, string][];
 }
 
+export interface CreateAlertDefinitionPayload {
+  name: string;
+  region: string;
+  description?: string;
+  service_type: string;
+  engineOption: string;
+  resource_ids: string[];
+  severity: string;
+  criteria: MetricCriteria[];
+  triggerCondition: TriggerCondition;
+  sink_ids: string[];
+}
+export interface MetricCriteria {
+  metric: string;
+  aggregation_type: string;
+  operator: string;
+  value: number;
+  dimension_filters: DimensionFilter[];
+}
```

Comment:
Have to change it to reflect the latest changes of API specification, like `rules` , `rule_criteria`